hana/saptune.sls

```diff
@@ -0,0 +1,12 @@
+{%- from "hana/map.jinja" import hana with context -%}
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes if node.host == host and hana.saptune_solution is defined %}
```

Comment:
I don't know if I would check `hana.saptune_solution is defined`.
And if it's not, but we have the solution in the node. This wouldn't work.
In any case we would need: 
```
{% for node in hana.nodes if node.host == host and (hana.saptune_solution is defined or node.saptune_solution is defined) %}
```